src/Moq/Properties/Resources.resx

```diff
@@ -120,6 +123,35 @@
   <data name="FieldsNotSupported" xml:space="preserve">
     <value>Expression {0} involves a field access, which is not supported. Use properties instead.</value>
   </data>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
There shouldn't be any blank lines (65-67 and 126-154).

Perhaps I should explain why I am being so peculiar about whitespace. This is a file generated & updated by tooling. Upon the next run, these blank lines will very likely be removed by the resources editor, causing whitespace changes yet again. So whoever introduces whitespace changes here will cause them also for the next committer.